### DISCUSSION TITLE
AutoStar II compatible

### DISCUSSION THREAD

**Discussion by abetoret** [scriptorron/AutoSTAR_remote#4] [General]
Hi [scriptorron](https://github.com/scriptorron),

Finally, I've made a new fork of the project.

[https://github.com/abetoret/AutoStarII_Remote](https://github.com/abetoret/AutoStarII_Remote)

These adaptations make the application compatible with Meade AutoStar II.

Regards.

**Comment by scriptorron** [comment]
Hello abetoret,


Looks really cool what you did! Funny that both AutoStar versions have different commands for basic keys (scroll-up, scroll-down, goto). I am thinking about merging both in a single application. But there are 2 concerns:

1. Will a combined application be useful for the user? It is certainly very seldom that someone has both AutoStar II and AutoStar #497.
2. Mere serious: I do not have a AutoStar II and will not be able to support it. All changes I do for AutoStar II will be untested and potentially buggy. 

Maybe it is better when we leave it separate for now.


Best Regards
scriptorron

**Comment by abetoret** [comment]
Hi,

Don't worry, I can test your developments with AutoStar II. But I think it is very rare to have both controllers.

We can collaborate with the developments. It would be interesting to have a configuration file to save the ASCOM driver selected, the color of the LCD, etc. I intend to develop this functionality.

I will keep you updated.

Regards.
